### PR TITLE
Mahshid's homework

### PR DESCRIPTION
**Pull request by MahshidAzami**

### CODE REVIEW

File: week-1-practise/level-1/10-function-output-3.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 function concatenate(firstWord, secondWord, thirdWord) {
-    // Write the body of this function to concatenate three words together
+  return firstWord.concat(secondWord, thirdWord);
```
Comment (amaalali) [inline week-1-practise/level-1/10-function-output-3.js]: Good to use the fact that concat takes multiple parameters!

File: week-1-practise/level-1/4-logic-error-1.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 // The syntax for this function is valid but it has an error, find it and fix it.
 
 function trimWord(word) {
-  return wordtrim();
+  return word.trim();
```
Comment (amaalali) [inline week-1-practise/level-1/4-logic-error-1.js]: Great. Quick one to check, ‘word.trim’ returns a different type of JS object. Check it and see what it returns (you can use typeof to help)

File: week-1-practise/level-2/2-format-currency.js [hunk 3]
```diff
@@ -6,7 +6,9 @@
   Write a function that transforms numbers into the format £0.00
 */
 
-function formatCurrency() {}
+function formatCurrency(price) {
+  return "£" + price.toFixed(2);
```
Comment (amaalali) [inline week-1-practise/level-2/2-format-currency.js]: Good stuff! What happens if someone used this function with a string? Try it out :)

File: week-1-practise/level-1/10-function-output-3.js (hunk 1 above)
Comment (MahshidAzami) [reply week-1-practise/level-1/10-function-output-3.js]: Thanks and sorry for delay in responding. Actually I thought we will be in touch just through the slack. I didn't notice your comment on my GitHub.

File: week-1-practise/level-1/4-logic-error-1.js (hunk 2 above)
Comment (MahshidAzami) [reply week-1-practise/level-1/4-logic-error-1.js]: I've checked, word.trim return string as I expected. so I think it won't change the type of object.

File: week-1-practise/level-2/2-format-currency.js (hunk 3 above)
Comment (MahshidAzami) [reply week-1-practise/level-2/2-format-currency.js]: I'm not sure I understand your question.

**Pull request closed without merging**